frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -435,22 +435,29 @@ const ReceiverWrapper: React.FC<ReceiverFormsWrapperProps> = React.memo(({ obj,
       setLoadError({ message: `Error alertManagerBaseURL not set` });
       return;
     }
-    coFetchJSON(`${alertManagerBaseURL}/api/v2/status/`).then((data) => {
-      const originalAlertmanagerConfigJSON = data?.config?.original;
-      if (_.isEmpty(originalAlertmanagerConfigJSON)) {
-        setLoadError({ message: 'alertmanager.v2.status.config.original not found.' });
-      } else {
-        try {
-          const { global } = safeLoad(originalAlertmanagerConfigJSON);
-          setAlertmanagerGlobals(global);
-          setLoaded(true);
-        } catch ({ message }) {
-          setLoadError({
-            message: `Error parsing Alertmanager config.original: ${message || 'invalid YAML'}`,
-          });
+    coFetchJSON(`${alertManagerBaseURL}/api/v2/status/`).then(
+      (data) => {
+        const originalAlertmanagerConfigJSON = data?.config?.original;
+        if (_.isEmpty(originalAlertmanagerConfigJSON)) {
+          setLoadError({ message: 'alertmanager.v2.status.config.original not found.' });
+        } else {
+          try {
+            const { global } = safeLoad(originalAlertmanagerConfigJSON);
+            setAlertmanagerGlobals(global);
+            setLoaded(true);
+          } catch ({ message }) {
+            setLoadError({
+              message: `Error parsing Alertmanager config.original: ${message || 'invalid YAML'}`,
+            });
+          }
         }
-      }
-    });
+      },
+      (reason) => {
```

Comment:
Use `coFetchJSON(...).then(...).catch(...)` instead

Reply:
fixed

Reply:
Hmm...I could just do `.catch((e) => setLoadError(e));`, which would result in showing just the following on the page (after one hits the [Create Receiver] button):
![image](https://user-images.githubusercontent.com/12733153/73389804-f0c81100-42a2-11ea-88ee-04843b2772a9.png)

Which would match what one sees in Monitoring -> Alerts | Silences:

![image](https://user-images.githubusercontent.com/12733153/73389874-0f2e0c80-42a3-11ea-9f51-aa715ca797c7.png)

idk, kinda feel like the error message `Error Loading Alertmanager Globals: Error loading /api/alertmanager/api/v2/status/: Error: Forbidden` is more descriptive.